src/main/java/core/basesyntax/FigureInterface.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureInterface {
+    double getFigureArea();
```

Comment:
```suggestion
    double getArea();
```

Мы можем импелементировать этот интерфейс не только для фигур, а и для других объектов, которые имеют площадь